images/bootstrap/containers.conf

```diff
@@ -4,6 +4,7 @@ userns="host"
 ipcns="host"
 cgroupns="host"
 cgroups="disabled"
+default_capabilities = ["CHOWN",  "DAC_OVERRIDE", "FOWNER",  "FSETID",  "KILL",  "NET_BIND_SERVICE",  "SETFCAP",  "SETGID",  "SETPCAP",  "SETUID", "SYS_CHROOT"]
```

Comment:
How can we make sure that we are adding only the capabilities we actually require?

@xpivarc @jean-edouard can we even do this?

Reply:
Trial and error. I am afraid I don't have a better answer.